core/controllers/collection_editor.py

```diff
@@ -28,6 +28,28 @@
 current_user_services = models.Registry.import_current_user_services()
 
 
+def get_matching_exploration_dicts(query_string, search_cursor):
+    """Given a query string and a search cursor, returns a list of exploration
```

Comment:
Let's follow the new docstring format that we're moving towards. See [here](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) and [here](https://github.com/oppia/oppia/pull/2465/files).